src/common/low_precision_transformations/src/network_helper.cpp

```diff
@@ -1999,6 +2001,13 @@ void NetworkHelper::insertDequantizationAfter(
         }
     }
 }
+
+ov::Output<ov::Node> NetworkHelper::getOutputWithSingleConsumer(const ov::Output<ov::Node>& output) {
+    auto node = output.get_node_shared_ptr();
+    if (node->get_output_size() > 1 || node->output(0).get_target_inputs().size() == 1)
+        return output;
+    return node->clone_with_new_inputs(node->input_values())->output(0);
+}
```

Comment:
Can we significant simplify the method to support `Constant` ops only? For this: 
1. rename to `getSingleConstant`
2. simplify:
```
ov::Output<ov::Node> NetworkHelper::getSingleConstant(const ov::Output<ov::Node>& output) {
    const auto node = output.get_node();
    if (!ngraph::is_type<Constant>(node)) {
        // throw exception here
    }

   if (output.get_target_inputs().size() == 1)
        return output;

    return node->clone_with_new_inputs(node->input_values())->output(0);
}
```